config/provider.go

```diff
@@ -139,10 +139,11 @@ func bumpVersionsWithEmbeddedLists(pc *ujconfig.Provider) {
 		// we would like to set the storage version to v1beta1 to facilitate
 		// downgrades.
 		r.SetCRDStorageVersion("v1beta1")
+		r.ControllerReconcileVersion = "v1beta1"
 		r.Conversions = []conversion.Conversion{
 			conversion.NewIdentityConversionExpandPaths(conversion.AllVersions, conversion.AllVersions, []string{"spec.forProvider", "spec.initProvider", "status.atProvider"}, r.CRDListConversionPaths()...),
```

Comment:
nit: We can also update the string literals for the three API roots here. But let's keep the tested state and we can do this in a follow-up PR if you also prefer.